cmd/osv-scanner/main_test.go

```diff
@@ -47,6 +47,10 @@ func dedent(t *testing.T, str string) string {
 	re = regexp.MustCompile(`^\r?\n`)
 	str = re.ReplaceAllString(str, "")
 
+	// 5. Remove trailing whitespace.
+	re = regexp.MustCompile(`\r?\n$`)
```

Comment:
I want to say that there was a good reason I didn't have this, like that otherwise you could output a bunch of junk at the end that'd look weird in peoples terminals, but I can't remember that off the top of my head so it's probably fine 🤷 

(I might stick this in the `osv-detector` tests later this week to see if anything changes to try and jog my memory)

Reply:
<details>
<summary>
@another-rex so I tested this in the `osv-detector` codebase and confirmed I intentionally left it out
</summary>

```
go test ./... -parallel=4
--- FAIL: TestRun_EmptyDirExitCode (0.00s)
    --- FAIL: TestRun_EmptyDirExitCode/#03 (0.00s)
        main_test.go:266: stdout
             got:
            Loaded the following OSV databases:
            
            
             want:
            Loaded the following OSV databases:
    --- FAIL: TestRun_EmptyDirExitCode/#04 (0.00s)
        main_test.go:266: stdout
             got:
            Loaded the following OSV databases:
            
            
             want:
            Loaded the following OSV databases:
--- FAIL: TestRun_ParseAs_CsvRow (0.00s)
    --- FAIL: TestRun_ParseAs_CsvRow/#02 (0.00s)
        main_test.go:676: stdout
             got:
            Loaded the following OSV databases:
            
            
             want:
            Loaded the following OSV databases:
--- FAIL: TestRun_Lockfile (0.00s)
    --- FAIL: TestRun_Lockfile/#04 (0.21s)
        main_test.go:444: stderr
             got:
            Loaded the following OSV databases:
              npm (2700 vulnerabilities, including withdrawn - last updated Mon, 28 Nov 2022 02:58:48 GMT)
            
            fixtures/locks-one/yarn.lock: found 1 package
              Using db npm (2700 vulnerabilities, including withdrawn - last updated Mon, 28 Nov 2022 02:58:48 GMT)
            
            
             want:
            Loaded the following OSV databases:
              npm (%% vulnerabilities, including withdrawn - last updated %%)
            
            fixtures/locks-one/yarn.lock: found 1 package
              Using db npm (%% vulnerabilities, including withdrawn - last updated %%)
--- FAIL: TestRun_ParseAs_CsvFile (0.00s)
    --- FAIL: TestRun_ParseAs_CsvFile/#01 (0.00s)
        main_test.go:721: stdout
             got:
            Loaded the following OSV databases:
            
            
             want:
            Loaded the following OSV databases:
--- FAIL: TestRun_ParseAs (0.00s)
    --- FAIL: TestRun_ParseAs/#02 (0.00s)
        main_test.go:597: stdout
             got:
            Loaded the following OSV databases:
            
            
            fixtures/locks-empty/composer.lock: found 0 packages
            
              no known vulnerabilities found
            
            
             want:
            Loaded the following OSV databases:
            
            
            fixtures/locks-empty/composer.lock: found 0 packages
            
              no known vulnerabilities found
FAIL
FAIL	github.com/g-rath/osv-detector	1.828s
?   	github.com/g-rath/osv-detector/internal	[no test files]
ok  	github.com/g-rath/osv-detector/internal/configer	(cached)
ok  	github.com/g-rath/osv-detector/internal/reporter	(cached)
ok  	github.com/g-rath/osv-detector/pkg/database	(cached)
ok  	github.com/g-rath/osv-detector/pkg/lockfile	(cached)
ok  	github.com/g-rath/osv-detector/pkg/semantic	(cached)
FAIL
```
</details>

I don't think you should make this change, but obviously your call :)

Reply:
Thanks for pointing it out! Removed those lines.